pom.xml

```diff
@@ -47,35 +48,36 @@
     <dependency>
       <groupId>org.apache.maven</groupId>
       <artifactId>maven-plugin-api</artifactId>
-      <version>3.3.9</version>
+      <version>${maven.version}</version>
+    </dependency>
+    <dependency>
+      <groupId>org.apache.maven</groupId>
+      <artifactId>maven-core</artifactId>
+      <version>${maven.version}</version>
     </dependency>
-
-    <!-- dependencies to annotations -->
     <dependency>
       <groupId>org.apache.maven.plugin-tools</groupId>
       <artifactId>maven-plugin-annotations</artifactId>
       <version>3.4</version>
       <scope>provided</scope>
     </dependency>
-    <dependency>
-      <groupId>org.apache.maven</groupId>
-      <artifactId>maven-core</artifactId>
-      <version>3.3.9</version>
-    </dependency>
+
     <dependency>
       <groupId>com.google.cloud.tools</groupId>
       <artifactId>app-tools-lib-for-java</artifactId>
       <version>0.1.0-SNAPSHOT</version>
     </dependency>
+
     <dependency>
```

Comment:
Guava doesn't provide a method to delete a directory recursively. It used to, but the new versions don't.